### PR TITLE
basic list, list item and initial details view

### PR DESCRIPTION
**Pull request by jellyfish-tom**
Screenshot 2023-01-24 at 15 28 15

- list
- clickable list item (for now logging item name)
- initial details view

### CODE REVIEW

File: src/screens/WeatherDetails/WeatherDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        marginRight: 10
+    },
+    image: {
+        width: 50,
+        height: 50
+    }
+});
+
+export default WeatherDetails;
+
```
Comment (jellyfish-tom) [inline src/screens/WeatherDetails/WeatherDetails.tsx:82]: still need to add prettier config and turn on save formatting

File: src/screens/WeatherDetails/WeatherDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { StyleSheet, View, Text, Image } from 'react-native';
+
+interface WeatherData {
+    key: string,
```
Comment (jellyfish-tom) [inline src/screens/WeatherDetails/WeatherDetails.tsx:5]: when I ll know shape of of object returned from that weather api will replace that and possibly move those files to some types folders if there will be need to share them

File: src/screens/WeatherDetails/WeatherDetails.tsx (hunk 1 above)
Comment (adamTrz) [reply src/screens/WeatherDetails/WeatherDetails.tsx:82]: Yup :D

File: src/screens/WeatherDetails/WeatherDetails.tsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                    <Text style={styles.text}>{item.city}</Text>
+                    <Text style={styles.text}>{item.county}</Text>
+                </View>
+                <Image source={{ uri: item.image }} style={styles.image} />
+                </View>
+                <View style={styles.bottomContainer}>
+                <View style={styles.textContainer}>
+                    <Text style={styles.label}>Temperature:</Text>
+                    <Text style={styles.text}>{item.temperature}</Text>
+                </View>
```
Comment (adamTrz) [inline src/screens/WeatherDetails/WeatherDetails.tsx:38]: Guess this could be moved to separate component since it repeats three times already? ;)

**Review by adamTrz — APPROVED** [review]

File: src/screens/WeatherDetails/WeatherDetails.tsx (hunk 3 above)
Comment (jellyfish-tom) [reply src/screens/WeatherDetails/WeatherDetails.tsx:38]: as an interesting thing - this whole code has been written by Chat GPT as an experiment of mine. Works quite fine, but sure, I will also refactor it before actually finishing :))

**Pull request merged**